### DISCUSSION TITLE
external viewer command for sioyek not working

### DISCUSSION THREAD

**Discussion by goyalyashpal** [sumatrapdfreader/sumatrapdf#3193] [Q&A] [answered]
* the command `sioyek "file.pdf" --page 7` works when triggered from terminal
* but the following setup in sumatra ain't working
* so, if some working solution ain't found, then this is most likely an issue (rather than a discussion)

References:
* sumatra/docs: [external-viewers](https://www.sumatrapdfreader.org/docs/Customize-external-viewers)
* https://github.com/ahrm/sioyek/discussions/535#discussioncomment-4441718

Versions:
* Sumatra: v3.4.6 64-bit
* os: win 10 10h2
* sioyek: 1.5.0

```
ExternalViewers [
	[
		CommandLine = "<some-location>\sioyek-release-windows\sioyek.exe" "%l" --page %p
		Name = Sioyek
		Filter = *.pdf
	]
]
```

**Comment by GitHubRulesOK** [comment, marked as answer]
I find its a bit odd in console but correct call should be \sioyek.exe --page 4 tutorial.pdf with all the usual windows cmd requirements

![image](https://user-images.githubusercontent.com/12754913/209454886-cbed4e0d-608d-4970-aa68-33b921b528a2.png)

***the problem from  your cut and paste that looks like `"%l"` it should be "%1"***

```
ExternalViewers [
[
CommandLine = "....whatever..\sioyek-release-windows\sioyek.exe" --page %p "%1"
Name = Sioye&k
Filter = *.pdf
]
]
```

**Reply by goyalyashpal** [reply]
> with all the usual windows cmd requirements

i am using msys2's `bash` shell


`l1` :( HOw siMilaR iS ThEiR reNdeRinG????
